pyart/core/transforms.py

```diff
@@ -78,6 +78,35 @@ def antenna_to_cartesian(ranges, azimuths, elevations):
     return x, y, z
 
 
+def cartesian_to_antenna(x, y, z):
+    """
+    Returns antenna coordinates from Cartesian coordinates.
+
+    Parameters
+    ----------
+    x, y, z : array
+        Cartesian coordinates in meters from the radar.
+
+    Returns
+    -------
+    ranges : array
+        Distances to the center of the radar gates (bins) in m.
+    azimuths : array
+        Azimuth angle of the radar in degrees. [-180., 180]
+    elevations : array
+        Elevation angle of the radar in degrees.
+    ke : float, optional
```

Comment:
I don't see ke in the returns list, did we want to remove it from the returns doc?